tools/wake-format/types.h

```diff
@@ -44,6 +44,7 @@ struct ctx_t {
   // 1: By default, don't explode unless you have to
   // 2: if prefer_explode, explode if you can
   bool prefer_explode = false;
+  bool nested_binop = false;
```

Comment:
Do we need both of these? I forgot the discussion around all this but I'm wondering if nested_binop can replace prefer_explode?

Reply:
Yeah, prefer_explode is more general and influences things more broadly. nested_binop just says "every child node from here on out is inside of at least one other binop" which turns out to be useful signal for many different formatting decisions

Reply:
What are those formatting decisions?